packages/entity-server/src/routes/hierarchy/middleware/attachCommonContext.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
All of this is copied out from the previously named `packages/entity-server/src/routes/hierarchy/middleware/middleware.ts`